src/main/java/frc/team2412/robot/subsystem/ClimbSubsystem.java

```diff
@@ -112,24 +150,30 @@ public void stopAngledArm() {
 
     public void setMotor(double position, WPI_TalonFX motor) {
         if (state == ClimbSubsystemState.ENABLED) {
-            motor.set(ControlMode.Position, position);
+            System.out.println("motor set: " + position);
+            //motor.set(ControlMode.Position, position)   >:( bad
+            motor.set(ControlMode.PercentOutput, 1);
```

Comment:
Note that this will make the motor run at full speed, which may not be desirable (I think this was leftover from testing in the simulator).

Consider reading the speed from shuffleboard or adding back the position control mode.